### DISCUSSION TITLE
Post-install OCC Commands

### DISCUSSION THREAD

**Discussion by FlattusBlastus** [nextcloud/all-in-one#2215] [Questions] [answered]
I install most of the apps at install time. After install, I execute the below commands. Any suggestions?

docker exec --user www-data -it nextcloud-aio-nextcloud php occ config:system:set default_phone_region --value="US"

docker exec --user www-data -it nextcloud-aio-nextcloud php occ memories:video-setup

docker exec --user www-data -it nextcloud-aio-nextcloud php occ memories:index

docker exec --user www-data -it nextcloud-aio-nextcloud php occ recognize:download-models

docker exec --user www-data -it nextcloud-aio-nextcloud php occ preview:generate-all -vvv

**Comment by szaimen** [comment, marked as answer]
Looks fine
